docs/examples.rst

```diff
@@ -3,8 +3,8 @@ Simple tests
 
 Ensure your device works with this simple test.
 
-.. literalinclude:: ../examples/motor_servo_sweep.py
-    :caption: examplesmotor_servo_sweep.py
+.. literalinclude:: ../examples/motor_servo_sweep_simpletest.py
+    :caption: examplesmotor_servo_sweep_simpletest.py
```

Comment:
There was a typo in the original. Please add a `/` to the `:caption:` so it reads 
```
:caption: examples/motor_servo_sweep_simpletest.py
```